tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -182,6 +180,22 @@ export class GraphExecutor implements FunctionExecutor {
         this.graph, this.weightMap, executionInfo);
   }
 
+  private cloneTensorMap(tensorsMap: NamedTensorsMap) {
+    return Object.fromEntries(
+        Object.entries(tensorsMap).map(([name, tensorsList]) => {
+          return [
+            name, tensorsList.map(tensor => {
+              if (tensor == null) {
+                return null;
+              }
+              const clone = tensor.clone();
+              keep(clone);
```

Comment:
Nit: Adding some docs on why we `keep` the cloned tensor.
```suggestion
               // Keep the clone because `model.execute()` may be called within
               // a `tidy()`, but the user may inspect these tensors after the tidy.
              keep(clone);
```